htsget-http-actix/src/lib.rs

```diff
@@ -3,55 +3,30 @@ use std::sync::Arc;
 
 use actix_web::web;
 
-// Async
-#[cfg(feature = "async")]
-use crate::handlers::{get, post, reads_service_info, variants_service_info};
-#[cfg(feature = "async")]
-use htsget_search::htsget::from_storage::HtsGetFromStorage;
-#[cfg(feature = "async")]
-use htsget_search::htsget::HtsGet;
-
-// Blocking
-#[cfg(not(feature = "async"))]
-use crate::handlers::blocking::{get, post, reads_service_info, variants_service_info};
+use htsget_config::config::HtsgetConfig;
+use htsget_config::regex_resolver::RegexResolver;
 #[cfg(not(feature = "async"))]
 use htsget_search::htsget::blocking::from_storage::HtsGetFromStorage;
 #[cfg(not(feature = "async"))]
 use htsget_search::htsget::blocking::HtsGet;
-
-use htsget_id_resolver::RegexResolver;
-
+#[cfg(feature = "async")]
+use htsget_search::htsget::from_storage::HtsGetFromStorage;
+#[cfg(feature = "async")]
+use htsget_search::htsget::HtsGet;
 #[cfg(not(feature = "async"))]
 use htsget_search::storage::blocking::local::LocalStorage;
 #[cfg(feature = "async")]
 use htsget_search::storage::local::LocalStorage;
 
-use crate::config::Config;
+// Async
+#[cfg(feature = "async")]
+use crate::handlers::{get, post, reads_service_info, variants_service_info};
+// Blocking
```

Comment:
Is there still blocking version(s) in the mix? Shall we get rid of it?

Reply:
Yeah the blocking code is still there. I'll make another PR to get rid of it.